app/assets/javascripts/ui_components/toolbar.coffee

```diff
@@ -0,0 +1,10 @@
+$ ->
+  # all checked checkboxes display as active
+  $(".toolbar .checkbox input[type='checkbox']:checked").parents('.checkbox').addClass('active')
```

Comment:
Usually I prefer CSS-only solutions over some jquery magic. Can't we just add `input[type="checkbox"]:checked` css selectors in the css file instead of dynamically adding a class?